nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/log/ComputationPool.java

```diff
@@ -82,6 +82,10 @@ public String getComputationName() {
 
     @SuppressWarnings("FutureReturnValueIgnored")
     public void start() {
+        if (threads == 0) {
+            log.info(metadata.name() + ": Empty pool");
```

Comment:
Nitpick: use Log4j2 and `log.info("{}: Empty pool", metadata::name);`

Reply:
For now, nuxeo-stream lib is using log4j 1, mixing both just creates problems when integrating this lib in other projects. But I agree that at some point, the upgrade to log4j 2 needs to be done.